notebooks/en/enterprise_cookbook_argilla.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>Just to be extra clear :)</p><p> </p><p>"Sending the instructions to the HF Text Generation Inference API</p><p> </p><p>Now we can send the instructions to the Text Generation Inference (TGI) APIs for both LLMs to get outputs we can evaluate."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
good point, updated the text a bit